scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,30 @@ setup_args() {
   fi
 }
 
+check_st2_host() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
+
+  # NOTE: lsof restricts the number of ports specified with "-i" to 100.
+  echo "Checking if required TCP ports are already in use."
+  echo ""
+  sudo lsof -V -P -i :80 -i :443 -i :4369 -i :5672 -i :9100 -i :9101 -i :9102 -i 25672 -i :27017 | grep LISTEN
+  if [ $? -eq 0 ]; then
+    echo "Not all required TCP ports are available."
```

Comment:
I think this could be a bit more helpful.
Ideally to answer 2 questions in error message: "what" happened and "how" to fix it.